README.md

```diff
@@ -79,14 +75,14 @@ export const handler = apiViewHandler(__filename, AlbumApi)
 
 ```typescript
 import { HttpMethod } from "@aws-cdk/aws-apigatewayv2"
-import { Route, ApiEvent } from "@jetkit/cdk"
+import { Lambda, ApiEvent } from "@jetkit/cdk"
 
 // a simple standalone function with a route attached
 export async function topSongsHandler(event: ApiEvent) {
   return "top songs"
 }
 // define route and lambda properties
-Route({
+Lambda({
```

Comment:
I find weird that the lambda still has a `path` and `methods` property, instead of having the result Lambda object being used in a Route declaration, something like this:

```typescript
const topSongsLambda = Lambda(handler, memorySize: 124, ...);
topSongsLambda.addEventSource(...) // Events from S3, SNS, SQS, etc...

const getTopSongs = Route(withLambda: topSongsLambda, path: "/top-songs", methods: [HttpMethod.GET]) ; // Handlers LambdaProxyIntegration internally
```

Reply:
Yeah that approach would certainly work as well for sure. The `.addEventSource()` bit feels like CDK more. Of course I think you _can_ do that with CDK if you need to hook up less common events to your Lambda. Will need to try it out.
My thoughts were to make the common case easy, and you need a Lambda for the route to work anyway, and we could simply add other common even sources as config keys to `Route()`, e.g.
```typescript
Lambda({ schedule: "10 * * * * *" })(function cronJob() { .... })
```

Reply:
It definitly can, I used this doc to base my comment on <https://docs.aws.amazon.com/cdk/api/latest/docs/aws-lambda-event-sources-readme.html>.

Also, you don't need a lambda for the route, it can work with those types: `HttpAlbIntegration, HttpNlbIntegration, HttpProxyIntegration, HttpServiceDiscoveryIntegration, LambdaProxyIntegration` as documented here: <https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-apigatewayv2.IHttpRouteIntegration.html>

I do agree it makes sense for a cron job to be configureable in the Lambda constructor, as the cronjob is a feature of lambda itself, I think it could also havea attribute to plug event sources automatically:

```typescript
Lambda(cronJob: { schedule: "10 * * * * *" },  eventSources: [ ... ] )(() => { .... })
```

Personally, the tools I enjoy the most are the ones that doesnt make the uncommon case harder to use then the common ones ( I'm not saying that jetkit makes uncommon cases harder to use, just sharing my overall opinion about tooling )